lib/ValidateDirectiveVisitor.ts

```diff
@@ -317,7 +317,7 @@ const validateEntryValueThrowing = <TContext>(
     type = type.ofType;
   }
 
-  if (value === null) {
+  if (value === null || value === undefined) {
```

Comment:
I think it’s better to return `value`, no?

Reply:
indeed. I'm sorry.